freedomofpress/securedrop-client
# update current user info during sync

# Description

Towards https://github.com/freedomofpress/securedrop-client/issues/76

Next step (after this PR): update gui auth widget to show updated name

# Test Plan

1. log in
2. change first and last name of the account you're logged into from the admin UI
3. wait until next sync then check local client db
   - [ ] verify local db shows updated first and last name
4. change first and last name to empty string
5. wait until next sync then check local client db
   - [ ] verify local db shows updated first and last name
6. change first and last name from journalist UI (not admin UI)
7. wait until next sync then check local client db
   - [ ] verify local db shows updated first and last name
8. change username from admin UI (can't do this from journalist UI)
   - [ ] verify local db shows updated username
